docs-requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Packages needed to build docs on https://readthedocs.org
-Django==1.11.10
+Django==2.2.9
```

Comment:
I DMed you an internal manual testing checklist we use -> Happy to run through it live with you, or to test it if you DM me a dev/staging link. Thanks for doing this!